yaksh/views.py

```diff
@@ -3237,3 +3237,51 @@ def download_course_progress(request, course_id):
     for student in stud_details:
         writer.writerow(student)
     return response
+
+
+@login_required
+@email_verified
+def download_course_yaml(request, course_id):
+    user = request.user
+    course = Course.objects.get(id=course_id)
```

Comment:
```course_id``` can be invalid then it make break.
Better to use ```get_object_or_404```

Reply:
Done